magma/magma
# fix(nms): Repair hot module replacement

## Summary

Renable hot module replacement using [react fast refresh](https://javascript.plainenglish.io/react-fast-refresh-the-new-react-hot-reloader-652c6645548c).
Also make sure that this work when upgrading to webpack 5. 

- fixes #13496

## Test Plan

- rebuild the docker container
- open the app in the browser
- change some code
   - make sure the code changes are reflected in the browser without reloading
- break the code
  - make that an error page is shown
- repair the code
  - make sure that error page disappears
